### PR TITLE
Random URL generation(Close #17)

### LINKED ISSUE #17: Random URL generation
URL generation for something like an authentication alternative.

Add an option to generate URLs like:

```
https://host:8080/nairj3gnw5ja39r23
```

### PR DESCRIPTION
**Pull request by shoz**

### CODE REVIEW

**Comment by mattn** [comment]
@shoz prob, you need to resolve conflicts.

**Comment by yudai** [comment]
Thank you for the PR.
Could you format your code as well?

**Comment by shoz** [comment]
Fixed!

File: main.go [hunk 1]
```diff
@@ -37,6 +37,11 @@ func main() {
 			Usage:  "Credential for Basic Authentication (ex: user:pass)",
 			EnvVar: "GOTTY_CREDENTIAL",
 		},
+		cli.BoolFlag{
+			Name:   "random-url, r",
+			Usage:  "Add a random string to the endpoint",
```
Comment (yudai) [inline main.go]: Could you add change to "Add a random string to the URL"?

**Comment by yudai** [comment]
Merged! Thanks!

**Pull request merged**